rack-protection/lib/rack/protection/content_security_policy.rb

```diff
@@ -62,7 +62,7 @@ def csp_policy
         # Set these key values to boolean 'true' to include in policy
         NO_ARG_DIRECTIVES.each do |d|
           if options.key?(d) && options[d].is_a?(TrueClass)
-            directives << d.to_s.sub(/_/, '-')
+            directives << d.to_s.gsub(/_/, '-')
```

Comment:
```suggestion
            directives << d.to_s.tr('_', '-')
```

Would it be possible to use `tr` to transliterate all of them from a String instead of a regular expression pattern?

Reply:
Yes, this is much better. Pushed the update.